spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -166,8 +164,6 @@ function onChanOpenInit(
   version: string) {
   // only ordered channels allowed
   abortTransactionUnless(order === ORDERED)
-  // only allow channels to "ibcaccount" port on counterparty chain
-  abortTransactionUnless(counterpartyPortIdentifier === "ibcaccount")
```

Comment:
Don't these handshakes still need to check counterparty port based on:
 
The interchain account module on a host chain must always bind to a port with the id `interchain_account`. Controller chains will bind to ports dynamically, with each port id set as `ics27-1-{owner-address}`.

Reply:
Yes updated.